dissect/target/tools/mount.py

```diff
@@ -51,12 +51,20 @@ def main():
         fname = f"disks/disk_{i}"
         vfs.map_file_fh(fname, d)
 
+    vnames = {}
     for i, v in enumerate(t.volumes):
-        fname = f"volumes/{v.name or f'volume_{i}'}"
-        vfs.map_file_fh(fname, v)
+        basename = f"{v.name or f'volume_{i}'}"
+        fname = basename
+
+        if fname in vnames:
```

Comment:
In theory this might fail with volumes like: 

```
- a
- a_1
- a

```
Because the 2nd a will try to claim a_1 without checking for that volume name right? 

Simple test code:
```

vnames = {}

class Vol:
        def __init__(self,name):
                self.name = name

volumes = [Vol("la"),Vol("la_1"),Vol("la")]
for i, v in enumerate(volumes):
        basename = f"{v.name}"
        fname = basename
        if fname in vnames:
                fname = f"{basename}_{len(vnames[basename])}"

        vnames[v] = fname
        vnames.setdefault(basename, []).append(v)
        print(fname)
```

gives

```
la
la_1
la_1
```